### DISCUSSION TITLE
Longhorn replication

### DISCUSSION THREAD

**Discussion by grig0701** [longhorn/longhorn#6540] [Q&A] [answered]
Hi, I have 3 workers available. I want to use Longhorn together with the Percona Operator for MySQL solution. But face a problem that the replication mechanism in Longhorn takes up too much memory, that is, if I want to have 750 GB of storage for the database, I need to have 3 times more.

Is it possible to disable the replication mechanism? Database will provide high availability at its level and the replication mechanism will be redundant At the moment, I use the defaultClassReplicaCount: 1. But when expanding the database, I noticed that disk space is allocated in a random order without binding to pods. That is, disk space for pvc is allocated on worker 3 and is attached to the pod on worker two. 

Is it possible to create the disk on the node where the pod is located? or is it already the level of the operator for the database?

**Comment by c3y1huang** [comment, marked as answer]
Have you checked out the [default data locality setting](https://longhorn.io/docs/1.5.1/references/settings/#default-data-locality)?

**Reply by grig0701** [reply]
Hi, thanks for your reply. It helped!
